org.mixedrealitytoolkit.input/Interactors/Ray/MRTKRayInteractor.cs

```diff
@@ -157,14 +262,33 @@ public override bool isHoverActive
                     bool hoverActive = base.isHoverActive;
                     if (hoverActive)
                     {
-                        if (xrController is ArticulatedHandController handController)
+                        bool isPalmFacingAway = false;
+
+                        #pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Why are both warning disabled?

Reply:
I scratched my head too when I saw it as I thought one would be enough but apparently one is needed for the data type:
![image](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/84108471/f364cd45-e189-46b0-b07e-13d045103950)
and the other is needed for the instance of the datatype
![image](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/84108471/13b6402f-7765-4471-b48f-2d1d93fc2f59)